src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -78,6 +79,107 @@ def get_dsn_content(self, dataset_name):
+            "avgblk": 500,
+            "recfm": "FB",
+            "blksize": 400,
+            "lrecl": 80,
+            "dsntype":"LIBRARY"
+        }):
+
+        """
+        Create a sequential or partitioned dataset.
+        Parameters
+        ----------
+            dataset_name
+        Returns
+        -------
+        json
+        """
+        
+        for opt in options:
+            if opt == "volser":
+                if options[opt] == None:
+                    options[opt] == "zmf046"
+                elif options[opt] != "zmf046":
+                    raise KeyError
+            if opt == "unit":
+                if options[opt] == None:
+                    options[opt] = "3390"
+            if opt == "dsorg":
+                if options[opt] == None:
+                    options[opt] = "PS"
+                else:
+                    if options[opt] not in ("PO", "PS"):
+                        raise KeyError
+            if opt == "alcunit":
+                if options[opt] == None:
+                    options[opt] = "TRK"
+                else:
+                    if options[opt] not in ("CYL", "TRK"):
+                        raise KeyError
+            if opt == "primary":
+                if options[opt] == None:
+                    options[opt] = 10
+            if opt == "secondary":
+                if options[opt] == None:
+                    options[opt] = 5
+            if opt == "dirblk":
+                if options[opt] == None:
+                    options[opt] = 10
+            if opt == "avgblk":
+                if options[opt] == None:
+                    options[opt] = 500
+            if opt == "recfm":
+                if options[opt] == None:
+                    options[opt] = "FB"
+                else:
+                    if options[opt] not in ("F", "FB", "V", "VB", "U"):
+                        raise KeyError
+            if opt == "blksize":
+                if options[opt] == None:
+                    options[opt] = 400
+            if opt == "lrecl":
+                if options[opt] == None:
+                    options[opt] = 80
+            if opt == "dsntype":
+                if options[opt] == None:
+                    options[opt] = "LIBRARY"
+                else:
+                    if options[opt] not in ("BASIC", "EXTPREF", "EXTREQ", "HFS", "LARGE", "PDS", "LIBRARY", "PIPE"):
+                        raise KeyError
+            
+
+
+        option = {
+            "volser": options["volser"],
+            "unit": options["unit"],
+            "dsorg": options["dsorg"],
+            "alcunit": options["alcunit"],
+            "primary": options["primary"],
+            "secondary": options["secondary"],
+            "dirblk": options["dirblk"],
+            "avgblk": options["avgblk"],
+            "recfm": options["recfm"],
+            "blksize": options["blksize"],
+            "lrecl": options["lrecl"],
+            "dsntype": options["LIBRARY"]
+        }
```

Comment:
There are additional options that should be supported: `storclass`, `mgntclass`, `dataclass`, and `like` (see the Request Body section [here](https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-create-sequential-partitioned-data-set#CreateDataSet)).